store/src/app/products/components/products-list/products-list.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { Product } from 'src/app/interfaces/products.interface';
+
+import { ProductsService } from '../../services/products.service';
+
+@Component({
+  selector: 'app-products-list',
+  templateUrl: './products-list.component.html',
+  styleUrls: ['./products-list.component.scss'],
+})
+export class ProductsListComponent implements OnInit {
+
+  constructor(
+    private productsService: ProductsService
+  ) { };
+
+  products: Product[] = [];
```

Comment:
Properties shoud be above of constructor

Reply:
done